src/site/content/en/blog/hid-examples/index.md

```diff
@@ -69,20 +68,34 @@ just peripherals.
 
 ## Game controllers
 
-Next up is something fewer of you are likely to have. Sony's DualShock 4 is a
-wireless controller for PlayStation 4 game consoles.
+### PlayStation&nbsp;4 Wireless controller
+
+Next up is something fewer of you are likely to have. Sony's DualShock&nbsp;4 is a
+wireless controller for PlayStation&nbsp;4 game consoles.
 
-The DualShock 4 Demo uses WebHID to receive the raw input reports from the
-DualShock 4 and provides a high-level API to access the controller's gyroscope,
+The DualShock&nbsp;4 Demo uses WebHID to receive the raw input reports from the
+DualShock&nbsp;4 and provides a high-level API to access the controller's gyroscope,
 accelerometer, touchpad, button, and thumbstick inputs. It also supports rumble
 and setting the color of an RGB LED housed within the controller.
 
 **Author:** TheBITLINK<br/>
-**Demo:** [DualShock 4 Demo](https://thebitlink.github.io/WebHID-DS4/) ([Source](https://github.com/TheBITLINK/WebHID-DS4))
+**Demo:** [DualShock&nbsp;4 Demo](https://thebitlink.github.io/WebHID-DS4/) ([Source](https://github.com/TheBITLINK/WebHID-DS4))
+
+### Nintendo Switch Joy-Con controllers
+
+Play the Chrome dino 🦖 offline game by actually jumping with a Nintendo Switch Joy-Con
```

Comment:
```suggestion
🦖 Play the Chrome dino offline game by actually jumping with a Nintendo Switch Joy-Con
```

Reply:
I'm worried about how it sounds when a screen reader encounters this.

Reply:
Just tested:

- My original text: "Play the Chrome dino tyrannosaurus rex offline game".
- Your proposal: " Tyrannosaurus rex play the Chrome dino offline game".

I think the original version flows better.